Dockerfile

```diff
@@ -1,14 +1,23 @@
 FROM elicocorp/odoo:7.0
 MAINTAINER Elico Corp <contact@elico-corp.com>
 
+# Set OS timezone to China
+RUN ln -sf /usr/share/zoneinfo/Asia/Shanghai /etc/localtime
+
+# Set Odoo timezone to China (will be set at startup thanks to Odoo
+# parameter substitution)
+ENV ODOO_TIMEZONE=Asia/Shanghai
```

Comment:
In elicocorp/odoo OS & Odoo are defined to UTC. elicocorp/odoo-china will have Asia/Shanghai timezone for OS & Odoo which IMO is what one would expect by default.